Automattic/woocommerce-payments
# Update `@woocommerce/explat` to v2.3.0

Fixes #6094 

#### Changes proposed in this Pull Request
- Update `@woocommerce/explat` to v2.3.0
- Update dependencies doc.









#### Testing instructions





* Ensure JS and lint tests pass.
* Ensure type declaration is still valid after the update.

-------------------


**Post merge**
